deps/chakrashim/src/v8objecttemplate.cc

```diff
@@ -748,6 +748,20 @@ Local<Object> ObjectTemplate::NewInstance(Handle<Object> prototype) {
     }
   }
 
+  if (!objectTemplateData->className.IsEmpty()) {
+    jsrt::IsolateShim* iso = jsrt::IsolateShim::GetCurrent();
+    if (jsrt::DefineProperty(newInstanceRef,
+			     iso->GetToStringTagSymbolPropertyIdRef(),
+			     jsrt::PropertyDescriptorOptionValues::True, /* writable */
```

Comment:
More specifically, in node-v8 the objects don't have the `Symbol.toStringTag` set, so you can do anything you want to it.